docs/ci.md

```diff
@@ -31,15 +31,15 @@ You will need to specify the following variables in the environment for the
 cibuildwheel action:
 
 ```yaml
-      - name: Build wheels
-        uses: pypa/cibuildwheel@...
-        env:
-          CIBW_PRERELEASE_PYTHONS: True
-          CIBW_FREE_THREADED_SUPPORT: True
-          CIBW_BUILD: cp313t-${{ matrix.buildplat }}
-          # TODO: remove along with installing build deps in
-          # cibw_before_build.sh when a released cython can build numpy
-          CIBW_BUILD_FRONTEND: "pip; args: --no-build-isolation"
+  - name: Build wheels
+    uses: pypa/cibuildwheel@...
+    env:
+      CIBW_PRERELEASE_PYTHONS: true
+      CIBW_FREE_THREADED_SUPPORT: true
+      CIBW_BUILD: cp313t-${{ matrix.buildplat }}
+      # TODO: remove along with installing build deps in
+      # cibw_before_build.sh when a released cython can build numpy
+      CIBW_BUILD_FRONTEND: 'pip; args: --no-build-isolation'
```

Comment:
Similarly, it's worth double checking all these changes actually work. They probably do but I don't want to publish incorrect CI config snippets.

Reply:
https://cibuildwheel.pypa.io/en/stable/options/#free-threaded-support explicitly mentions that setting this to `true` works and goes on to give examples where the environment variable is set to 1.

About the string, the yaml file in the [`pandas` PR](https://github.com/pandas-dev/pandas/pull/59136/files) has quotes around it and it's okay.